wiki/People/Community_Contributors/en.md

```diff
@@ -212,4 +222,18 @@ Not to be mistaken as [osu! Alumni](/wiki/People/osu!_Alumni), which are previou
 | ::{ flag=TR }:: [frukoyurdakul](https://osu.ppy.sh/users/7612550) | Outstanding contribution to the osu!taiko mapping scene via utility development |
 | ::{ flag=BR }:: [LeoFLT](https://osu.ppy.sh/users/3668779) | Outstanding contribution to the osu! tournament scene and the World Cups |
 | ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251) | Outstanding contribution to the osu! tournament scene and the World Cups |
-| ::{ flag=NL }:: [cavoeboy](https://osu.ppy.sh/users/7361815) | Outstanding contribution to the osu! IRL events and tournament scene |
+| ::{ flag=NL }:: [cavoeboy](https://osu.ppy.sh/users/7361815) | Outstanding contribution to osu! IRL events and the tournament scene |
+
+## History
+
+During February 2018, the old contributor badge by [RBRat3](https://osu.ppy.sh/users/307202) was redesigned by [flyte](https://osu.ppy.sh/users/3103765) due to not matching the new website's colour scheme.[^redesign-reasons]
+
+![1st design change comparison](img/design-change-1.png)
+
+Due to a lack of "soul",[^redesign-reasons] the design was changed again on 21 July 2023. This time the initial design was refreshed by [RBRat3](https://osu.ppy.sh/users/307202) and split into two versions, where older contributors (with a user ID below 4,000,000) saw a revert to the original while newer contributors received a more colourful variant.
```

Comment:
country flags on all the user links unless there is some reason not to these days idk

Reply:
second RBRat mention also might not need a link at all since it's so soon after the first one, your call